docs/documentation/accessibility.md

```diff
@@ -0,0 +1,142 @@
+
+Below is a quick script that renders all of these colors:
+
+```python
+# /// script
+# dependencies = [
+#   "rich",
+# ]
+from rich.console import Console
+from rich.box import SIMPLE
+from rich.table import Table
+
+console = Console()
+table = Table(title="ANSI Colors", box=SIMPLE)
+colors = ["black", "red", "green", "yellow", "blue", "magenta", "cyan", "white"]
+table.add_column("Color", style="bold")
+
+for variant in ["Normal", "Dim", "Bright", "Dim +\nBright"]:
+    table.add_column(variant, style="bold")
+
+for color in colors:
+    table.add_row(
+        color,
+        *[
+            f"[{style}{color}]██████[/{style}{color}]"
+            for style in ["", "dim ", "bright_", "dim bright_"]
+        ]
+    )
+
+console.print(table)
+```
+
+The fact that the colors are not deterministic is a _benefit_ for accessibility; it means, for example, a user can customize their terminal so that the ANSI "red" is more suitable for them.
+Nearly every modern terminal allows for this sort of customisation.
+
+This means that developers looking to create a more accessible experience should prefer ANSI colors.
+
+So, for example:
+
+- `RichHelpConfiguration(style_option="red")` is more accessible because users can configure the hex value of this red.
+- `RichHelpConfiguration(style_option="#FF0000")` is less accessible because it is not configurable by the end user.
+
+## Colorblindness accessibility for users
+
+If you are a user of a **rich-click** CLI, there are a few options you have to improve accessibility for yourself.
+
+### 1. Configure your terminal's 4-bit ANSI colors
+
+The 4-bit ANSI color system is a set of 16 different colors implemented in effectively every terminal, and they are the most common way to set colors.
+These colors are **not** deterministic; different terminals use slightly different hex values for the ANSI colors. [Wikipedia has a full breakdown of all the variations in these colors](https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit)
+
+Most modern terminals have the ability to customise all of these colors in the terminals' settings.
+If you are having difficulty distinguishing colors, it is recommended that you adjust these settings.
+
+!!! note
+    This will only work for CLIs that utilize the 4-bit ANSI color system.
+    CLIs that utilize hex values or other color systems will not be impacted by your terminal's ANSI color settings.
+
+### 2. Use the `NO_COLOR` environment variable
+
+Rich uses the `NO_COLOR` standard ([more information here](https://no-color.org/)), which means it has a built-in capability that allows the user to suppress color.
+The Rich docs also go over additional and related console settings [here](https://rich.readthedocs.io/en/latest/console.html).
+
+So, instead of running your CLI like this:
+
+```shell
+$ python cli.py
+```
+
+You can instead do the following:
+
+```shell
+# Set environment variable, then run:
+export NO_COLOR=1
```

Comment:
Worth mentioning adding this to `.bashrc` or equivalent for folks who don't know about that.

Maybe even a tabbed example snippet for different shells with echo append commands? Could be overkill.

Reply:
Was thinking about that, good idea!

Reply:
Added in the comparison docs and changed this part to a reference to those docs instead, in 3abaaa27791432d778b3298d2d07f30ae2531588.